### PR TITLE
create user's top page and user edit page

### PR DESCRIPTION
**Pull request by yumeR07**
I create user's top page and user edit page.

By mistake, I attached an image...

### CODE REVIEW

File: resources/views/users/top.blade.php [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+    <div class="timetable mx-auto">
+      <h2 class="display-5">TIME TABLE</h2>
+        <div class="timetable-date fs-5">
+          <i class="fa-solid fa-chevron-left"></i> 2023/06/10 <i class="fa-solid fa-chevron-right"></i>
+        </div>
+          <table class="mx-auto mt-2">
+            <tr>
+              <th>10:00~10:45</th>
+              <td>a</td>
```
Comment (keanbillyfernandez) [inline resources/views/users/top.blade.php:43]: is this a permanent? or just a temporary?

File: app/Http/Controllers/ProfileController.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    public function update(Request $request){
+        $user = $this->user->findOrFail(Auth::user()->id);
+
+        $all_levels = $this->level->all();
+
+        $request->validate([
+            'avatar' => 'max:1048|mimes:jpeg,jpg,png,gif',
+            'level_id' => 'required',
+            'user_name' => 'required|max:50',
+            'comment' => 'max:100',
```
Comment (keanbillyfernandez) [inline app/Http/Controllers/ProfileController.php:47]: Pls make this atleast 250 character

File: resources/views/users/top.blade.php (hunk 1 above)
Comment (yumeR07) [reply resources/views/users/top.blade.php:43]: This is just a temporary.
This part is going to take a long time, so I'll merge the other parts.

File: app/Http/Controllers/ProfileController.php (hunk 2 above)
Comment (yumeR07) [reply app/Http/Controllers/ProfileController.php:47]: Modified as follows.
 
 $request->validate([
            'avatar' => 'max:1048|mimes:jpeg,jpg,png,gif',
            'level_id' => 'required',
            'user_name' => 'required|max:250',
            'comment' => 'max:250',
            'email' => 'required|max:250'
        ]);

File: routes/web.php [hunk 3]
```diff
@@ -22,13 +27,20 @@
...
 
 // Route Group
 Route::group(['middleware' => 'auth'], function () {
     # LOGINED USERS ONLY
+    Route::group(['prefix'=>'users','as'=>'users.'], function(){
+        Route::get('/', [HomeController::class, 'index'])->name('top');
+        Route::get('/reserved/show', [HomeController::class, 'show'])->name('reserved.show.details');
+        Route::get('/profile/{id}/show', [ProfileController::class, 'show'])->name('profile.show');
+        Route::get('/profile/edit', [ProfileController::class, 'edit'])->name('profile.edit');
+        Route::patch('/profile/update', [ProfileController::class, 'update'])->name('profile.update');
```
Comment (raymonkredo) [inline routes/web.php:42]: for `/profile/{id}/show`, change that to `/{user}`
for `/profile/edit`, change that to `/{user}/edit`
for `/profile/update`, change that to  just `/`

**Review by raymonkredo — CHANGES_REQUESTED** [review]

**Pull request merged**